plugins/parsers/opentsdb/parser.go

```diff
@@ -0,0 +1,127 @@
+package opentsdb
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/metric"
+	"github.com/influxdata/telegraf/plugins/parsers"
+)
+
+// Parser encapsulates a OpenTSDB Parser.
+type Parser struct {
```

Comment:
Renamed `OpenTSDBParser` -> `Parser` throughout to avoid the stutter in `opentsdb.OpenTSDBParser`. `opentsdb.Parser` should be fine, but just let me know if I should change this back.

Reply:
Seems to be done in other parsers as well, so it's fine.